### DISCUSSION TITLE
Using Go to definition (F12) and cmd click to jump to definition

### DISCUSSION THREAD

**Discussion by MacKlinkachorn** [coder/code-server#2955] [Q&A] [answered]
Is there any way we can click at the function/variable and jump to the file or line where it is declared? In vscode desktop, we can use F12 and cmd-click to examine any variable or function. Is there anything equivalent in code-server? Thanks!

**Comment by code-asher** [comment, marked as answer]
ctrl/cmd+click and ctrl/cmd+f12 are supposed to work but I tested on 3.9.1 and they seem sporadic; it worked for me in a Typescript file but not in a Python file.

They seem to work everywhere in latest main though, so this might be fixed in the next release.

**Reply by MacKlinkachorn** [reply]
Thank you!

**Comment by nskrjabins** [comment]
@code-asher I still have the same issue in version 4.9.1. The Go to Definition is not even an option when right clicking a function. Is there perhaps a setting that needs to be enabled? 

![2](https://user-images.githubusercontent.com/60360098/217603751-d7573016-4fd8-4bad-91df-2e2177a6f159.png)
![1](https://user-images.githubusercontent.com/60360098/217603772-cd217daf-464b-4435-853f-f890f3771ed0.png)

**Reply by code-asher** [reply]
I believe the only requirement is to install the Python extension.  No required settings as far as I am aware.
